src/python/install.sh

```diff
@@ -10,7 +10,7 @@
 PYTHON_VERSION=${VERSION:-"latest"} # 'system' checks the base image first, else installs 'latest'
 INSTALL_PYTHON_TOOLS=${INSTALLTOOLS:-"true"}
 OPTIMIZE_BUILD_FROM_SOURCE=${OPTIMIZE:-"false"}
-PYTHON_INSTALL_PATH=${INSTALL_PATH:-"/usr/local/python"}
+PYTHON_INSTALL_PATH=${INSTALLPATH}
```

Comment:
Is this same as 👇  ? 

> PYTHON_INSTALL_PATH=${INSTALLPATH:-"/usr/local/python"}

Reply:
We get defaults from the `devcontainer-feature.json` (set to `/usr/local/python` by default).  It seems confusing to me that in a lot of the scripts we have two defaults